README.md

```diff
@@ -93,7 +151,7 @@ the complete HTML document, you can use the `emogrifyBodyContent` instead:
 $bodyContent = $emogrifier->emogrifyBodyContent();
 ```
 
-## Options
+### Options
```

Comment:
I wonder if this should now even be a level 4 heading, to be clear it only applies to using the legacy Emogrifier class?

Reply:
Makes sense. Changed.